arches/management/commands/updateproject.py

```diff
@@ -64,3 +65,8 @@ def update_to_v7(self):
                         graphs.append(graphid)
                 management.call_command("graph", operation="publish", update_instances=True, graphs=",".join(graphs))
                 cursor.execute("drop table if exists temp_graph_status")
+
+    def update_to_v7_5(self):
+        # Republish to pick up CardsXNodeXWidget changes from
+        # migration 10150_add_default_value_file_list.py
+        management.call_command("graph", operation="publish")
```

Comment:
When I tested this it ended up publishing even unpublished graphs. There could be a reasons not to update the existing publications, but if we pass in the update flag we can prevent publishing unpublished models. 

I tried that just to confirm and found I got an integrity error if the graph was not already published, so I had to wrap it in a try block. 
```
from django.db.utils import IntegrityError

try:
    management.call_command("graph", operation="publish", update=True)
except IntegrityError:
    pass
```

Reply:
Ah got it, thanks. Added `--update` flag.

Re: the IntegrityError, I'd expect that to roll back the entire transaction. I'll see if I can do something about the root cause tomorrow?

Reply:
#10410